lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -374,10 +374,16 @@ class DetailsRenderer {
       for (const heading of headings) {
         const valueFragment = this._dom.createFragment();
 
-        const value = row[heading.key];
-        const valueElement =
-          value !== undefined && !Array.isArray(value) && this._renderTableValue(value, heading);
-        if (valueElement) valueFragment.appendChild(valueElement);
+        if (heading.key === '_') {
```

Comment:
oh I see this is a new thing! can we make this more explicit like 

```suggestion
        if (heading.key === '__empty-element-with-subrows-only__') {
```

not that literally but something that more obviously suggests what's going on?

Reply:
what about just `null`?

Reply:
warnings would get caught in a report test, right?